tests/test_schemas.py

```diff
@@ -40,21 +46,56 @@ def generate_graph_data_model(helpers, path_to_data_model):
 
 @pytest.fixture
 def DME(helpers, data_model_name='example.model.csv'):
-    path_to_data_model = helpers.get_data_path("example.model.jsonld")
-
-    graph_data_model = generate_graph_data_model(helpers, path_to_data_model=path_to_data_model)
+    '''
+    In future could pull using helpers.
+    '''
+    graph_data_model = generate_graph_data_model(helpers, data_model_name=data_model_name)
     DME = DataModelGraphExplorer(graph_data_model)
     yield DME
 
+
 class TestDataModelParser:
     def test_get_base_schema_path(self, helpers):
-        return
-
-    def test_get_model_type(self):
-        return
-
-    def test_parse_model(self):
-        return
+        '''Test that base schema path is returned properly.
+        Note:
+            data model parser class does not currently accept an new path to a base schema,
+            so just test that default BioThings data model path is returned.
+        '''
+        # Instantiate Data model parser.
+        data_model_parser = helpers.get_data_model_parser(data_model_name='example.model.csv')
+
+        # Get path to default biothings model.
+        biothings_path = data_model_parser._get_base_schema_path(base_schema=None)
+
+        assert os.path.basename(biothings_path) == "biothings.model.jsonld"
+
+    @pytest.mark.parametrize("data_model", list(DATA_MODEL_DICT.keys()), ids=list(DATA_MODEL_DICT.values()))
```

Comment:
Just like I mentioned in other PR, I think this part could be moved to the class level. Something like:
```
@pytest.mark.parametrize("data_model", list(DATA_MODEL_DICT.keys()), ids=list(DATA_MODEL_DICT.values()))
class TestDataModelParser:
```

So that we don't have to repeat this line for every single function